src/main/webapp/dashboards/patient.jsp

```diff
@@ -124,13 +121,13 @@
                                     <td>${appointment.type}</td>
                                     <td>
                                         <form method="get" action="${pageContext.request.contextPath}/appointments/update">
-                                            <button>Edit</button>
+                                            <button id="pinkButtonAlternative">Edit</button>
```

Comment:
The id will need to be changed to a class in the future. I've already made the change in my own branch so I can do that once this is merged.